fedimintd/src/envs.rs

```diff
@@ -1 +1,38 @@
+// Env variable to TODO
 pub const FM_DISABLE_META_MODULE_ENV: &str = "FM_DISABLE_META_MODULE";
+
+// Env variable to TODO
+pub const FM_EXTRA_DKG_META_VAR_ENV: &str = "FM_EXTRA_DKG_META";
```

Comment:
`_VAR_ENV` is not like other ones (`_VAR`).

Reply:
changed